cd/manager/jobs/anchor.go

```diff
@@ -9,8 +9,9 @@ import (
 	"github.com/3box/pipeline-tools/cd/manager"
 )
 
+// Allow up to 3 hours for anchor workers to run
 const AnchorStalledTime = 3 * time.Hour
-const TaskIdParam = "id"
+const AnchorTaskIdParam = "id"
```

Comment:
Mind adding this as a literal under this section in `models.go`?
```
const (
	JobParam_Component string = "component"
	JobParam_Sha       string = "sha"
	JobParam_Error     string = "error"
	JobParam_Layout    string = "layout"
	JobParam_Manual    string = "manual"
	JobParam_Force     string = "force"
	JobParam_Start     string = "start"
	JobParam_Rollback  string = "rollback"
	JobParam_Delayed   string = "delayed"
	JobParam_Stalled   string = "stalled"
	JobParam_Source    string = "source"
)
```
`id` is generic enough to be useful in other places, so maybe `JobParam_Id        string = "id"`.

Reply:
will do